src/waitress/runner.py

```diff
@@ -177,6 +177,8 @@
         detecting if a client closed the connection while its request is being
         processed. Default is 0.
 
+    --[no-]decode-path
+        Decode URL-escaped characters in the path. Enabled by default.
```

Comment:
This should be two different args, not "smooshed" into a single arg, one each for `--decode-path` and `--no-decode-path`.

Reply:
You might be right, but that's the pattern used throughout the file. If we change this one, we should change the rest too.

Reply:
How about this: instead of `--[no-]decode-path``, the argument line should say `--decode-path, --no-decode-path`. Similarly, `--[no-]ipv4` should change to `--ipv4, --no-ipv4`. There are 3 others.

Reply:
I implemented your suggestion (for all 5 instances of `--[no-]`) and liked the result.

Reply:
Thanks for taking the extra step. I was looking only at the diff in the PR, not the other instances of `[no-]`. These changes improve search results in the docs, too.